.circleci/config.yml

```diff
@@ -201,9 +201,7 @@ jobs:
           name: Run fMRIPrep tests
           no_output_timeout: 2h
           command: |
-            docker run -ti --rm=false \
-              --entrypoint="/usr/local/miniconda/bin/py.test" \
-              poldracklab/fmriprep:latest . \
+            py.test /home/circleci/src/fmriprep \
```

Comment:
Sorry, still meant run it through docker, just use `fmriprep` instead of the full path. So:

```Shell
docker run -ti --rm=false \
  --entrypoint="/usr/local/miniconda/bin/py.test" \
  poldracklab/fmriprep:latest fmriprep --doctest-modules
```

IDK. Might not work.

Reply:
okay, I misunderstood your comment above, sorry about that